src/main/java/edu/wpi/first/gradlerio/GradleRIOPlugin.java

```diff
@@ -56,6 +56,9 @@ interface Parameters extends FlowParameters {
         @Override
         public void execute(Parameters parameters) {
             Optional<Throwable> failure = parameters.getBuildResult().get().getFailure();
+            if (System.getProperty("user.dir").contains("OneDrive")) {
```

Comment:
We want to be checking the project directory, not the user directory. Because this would still error even if you move the project.

Reply:
Also, we'd want to do a case insensitive comparison.

Reply:
Both of these comments still need to be fixed.